UnrealPlugin/ThirdPersonMPCloudDeploy.md

```diff
@@ -91,56 +92,10 @@ This option runs your game server(s) in docker containers, isolating them from e
 * Select __Add Build__
 * Skip ahead to the __Configure Regions__ section below
 
-### Setting up a Linux Dedicated Server on PlayFab
-
-A more detailed Linux guide can be found [here](https://github.com/PlayFab/MpsAgent/blob/main/LocalMultiplayerAgent/MultiplayerSettingsLinuxContainersOnWindowsSample.json)
-
-During testing the following Dockerfile + startup.sh script worked excellent with the PlayFab Linux VM:
-
-#### The Linux Dockerfile:
-
-```Dockerfile
-FROM ubuntu:18.04
-
-# Unreal refuses to run as root user, so we must create a user to run as
-# Docker uses root by default
-RUN useradd --system ue
-USER ue
-
-EXPOSE 7777/udp
-
-WORKDIR /server
-
-COPY --chown=ue:ue . /server
-USER root
-CMD ./startup.sh
-```
-
-### startup.sh bash script:
-
-```bash
```

Comment:
this was deemed necessary so the Unreal process can write to /data/GameLogs. Did you try without it and saw that it works?

Reply:
@stkrwork instead of using an extra bash script, we're evaluating adding
```CMD chown -R ue.ue $PF_SERVER_LOG_DIRECTORY && su ue -c ./<projectname>Server.sh```
to the Dockerfile. We think it should work, WDYT?

Reply:
that could work